packages/richtext-lexical/src/features/indent/server/index.ts

```diff
@@ -1,10 +1,31 @@
 import { createServerFeature } from '../../../utilities/createServerFeature.js'
 import { i18n } from './i18n.js'
 
-export const IndentFeature = createServerFeature({
-  feature: {
-    ClientFeature: '@payloadcms/richtext-lexical/client#IndentFeatureClient',
-    i18n,
+export type IndentFeatureProps = {
+  /**
+   * The nodes that should not be indented. "type" property of the nodes you don't want to be indented.
+   * These can be: "paragraph", "heading", "listitem", "quote" or other indentable nodes if they exist.
+   */
+  disabledNodes?: string[]
+}
+
+/**
+ * This feature is used to indent and outdent the text in the editor.
+ *
+ * @param props - The props for the feature.
+ * @param props.disabledNodes - The nodes that should not be indented. "type" property of the nodes you don't want to be indented. These can be: "paragraph", "heading", "listitem", "quote" or other indentable nodes if they exist.
```

Comment:
These JSDocs are duplicative with the JSDocs defined in `IndentFeatureProps`. Do we need them?

If you remove them, the editor will still show them when hovering over `disabledNodes`